### DISCUSSION TITLE
Focus element when using qute-pass

### DISCUSSION THREAD

**Discussion by OmeletWithoutEgg** [qutebrowser/qutebrowser#7081] [Q&A] [answered]
When using the userscript qute-pass, sometimes I need to focus on the input field, and then run the script, because right after entering some websites I am not on the input field.
I tried to use "hint" to select the input element, and if I select an editable element, I will automatically enter insert mode. It would be convenient if I  could just use one key combination to auto fill in the field.
I found that this behavior is controlled by input.insert_mode.auto_enter.
However, in most cases I do like the input.insert_mode.auto_enter to be true. Is there any elegant way to get rid of that?

**Comment by The-Compiler** [comment, marked as answer]
I suppose you could bind a key combination to launch qute-pass from insert mode.
